resources/META-INF/plugin.xml

```diff
@@ -16,17 +16,12 @@
     <!--</change-notes>-->
 
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-    <idea-version since-build="145.0"/>
+    <idea-version since-build="107.105"/>
```

Comment:
I set **since-build** to same version like original [emacsIDEAs](https://github.com/whunmr/emacsIDEAs/blob/master/resources/META-INF/plugin.xml#L250)